packages/web-components/tests/snapshots/dds-locale-modal.md

```diff
@@ -64,10 +61,7 @@
 >
   <div class="bx--modal-content">
     <dds-expressive-modal-header>
-      <dds-expressive-modal-close-button
-        role="button"
-        tabindex="0"
-      >
+      <dds-expressive-modal-close-button>
```

Comment:
@asudoh I see in the snapshot that the role/tabindex is gone in the snapshot, wondering if this is intentional?

Reply:
Thank you @jeffchew for bringing this up! The answer is yes, it's intentional. Both of those features are moved to `<button>` in the shadow DOM of `<dds-expressive-modal-close-button>`.

Reply:
thanks, just confirming!

Side note on this, this means that the snapshots made are shallow? Wondering if we should do deeper shapshots?

Reply:
> Wondering if we should do deeper shapshots?

Only single DOM tree is supported by our toolchain, unfortunately. It means that we can generate only a single shadow DOM tree of a component, not nested (composed) tree.